dev/MRTCore/packaging/MrtCore.PriGen.targets

```diff
@@ -9,10 +9,9 @@
 
   <!--
     For .NET projects, we use the default file inclusion support that's built into the .NET SDK for the Windows App SDK. For that, we need to set these properties.
+    The support is only available in version 6.0.300 or higher.
   -->
-  <PropertyGroup Condition="'$(UsingMicrosoftNETSdk)' == 'true'">
-    <EnableDefaultContentItems Condition="'$(EnableDefaultContentItems)' == ''">true</EnableDefaultContentItems>
-    <EnableDefaultPRIResourceItems Condition="'$(EnableDefaultPRIResourceItems)' == ''">true</EnableDefaultPRIResourceItems>
+  <PropertyGroup Condition="'$(UsingMicrosoftNETSdk)' == 'true' and $([MSBuild]::VersionGreaterThanOrEquals($(NETCoreSdkVersion), '6.0.300'))">
```

Comment:
>and $([MSBuild]::VersionGreaterThanOrEquals($(NETCoreSdkVersion), '6.0.300')) [](http://example.com/codeflow?start=64&length=77)

this is not necessary

Reply:
True. Since it makes the intention explicit, I prefer to keep it. If you think we should remove it, I can do that.

Reply:
I'd remove it if it was me. But I'll leave it up to you.